reference/client-access/python-sdk.md

```diff
@@ -0,0 +1,287 @@
+
+```bash
+$ pip install libsql_client
+```
+
+## Initialization
+
+Import the libsql_client module:
+
+```py
+import libsql_client
+```
+
+If you want to use the synchronous API, use the `create_client_sync` function:
+
+```py
+client = libsql_client.create_client_sync(
+    url="libsql://your-database.turso.io"
+    auth_token="your-auth-token"
+)
+```
+
+If you want to use the asynchronous API, use the `create_client` function along
+with `asyncio` and async/await syntax:
+
+```py
+client = libsql_client.create_client_sync(
+    url="libsql://your-database.turso.io"
+    auth_token="your-auth-token"
+)
+```
+
+The `auth_token` parameter is only required when using a remote database
+instance managed by Turso.
+
+In either case, the returned Client object exposes an identical API. The
+synchronous client is a thin wrapper around the asynchronous client.
+
+The examples on this page use the synchronous API. The equivalent async code can
+be easily derived from that by applying `async` and `await`.  For example, a
+synchronous call to the client's `execute()` method takes this form:
+
+```py
+with client:
+    result_set = client.execute("...")
+    # Work with query results
+```
+
+While an asynchronous call takes this form:
+
+```py
+async with client:
+    result_set = await client.execute("...")
+    # Work with query results
+```
+
+In both cases above, the `with` keyword is used to close the client after the
+work is complete. If you want to work with the client object across functions,
+you should call `close()` on it after you're done with the client to clean up
+the resources it uses.
+
+## Example data set
+
+All of the examples in this section assume tables and data established by these
+statements:
+
+```sql
+create table example_users (
+    uid text primary key,
+    email text
+);
+create table example_scores (
+    uid text,
+    level integer,
+    score integer,
+    primary key (uid, level)
+);
+
+insert into example_users values ('uid1', 'foo@bar.com');
+insert into example_users values ('uid2', 'baz@bar.com');
+insert into example_scores values ('uid1', 1, 100);
+insert into example_scores values ('uid1', 2, 95);
+```
+
+## Execute a single statement
+
+### SQL string argument
+
+You can execute a single statement with the `execute()` method of a Client
+object. It returns a [ResultSet](#resultset) object containing the results of
+the query.
+
+```py
+rs = client.execute("select * from users")
+
+# rs.columns == ('uid', 'email')
+# rs.rows[0] == ('uid1', 'foo@bar.com')
+# rs.rows[1] == ('uid2', 'baz@bar.com')
+```
+
+### Positional placeholders
+
+libSQL supports positional placeholders using the same syntax as SQLite. Pass
+two parameters, with the first (`stmt`) being the SQL string with placeholders,
+and the second (`args`) being an array of values to bind to the placeholders.
+
+```py
+rs = client.execute(
+    stmt="select score from example_scores where uid = ? and level = ?",
+    args=[ "uid1", 2 ]
```

Comment:
It's not necessary to use named parameters here, it's more natural to write just `client.execute("select ...", ["uid1", 2])`.

Reply:
I was actually thinking that we should allow the `execute("statement", args)` syntax in the JS/TS client, too :) Using named arguments for a function that takes two arguments with obvious meaning looks very unnatural to me, but I don't mind if you use it in the docs. However, we should also show the positional variant, to make sure that people don't think that the use of named parameters is required.